core/src/main/java/dk/alexandra/fresco/lib/field/bool/BasicLogicFactory.java

```diff
@@ -44,9 +43,9 @@
  * 
  *
  */
+@Deprecated
```

Comment:
Go through all the interface and remove+delete the unused interfaces and methods. As far as I can tell it is every one.